maat-court-data-api/src/main/java/gov/uk/courtdata/repository/RepOrderMvoRepository.java

```diff
@@ -0,0 +1,10 @@
+package gov.uk.courtdata.repository;
+
+import gov.uk.courtdata.entity.RepOrderMvoEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+
+public interface RepOrderMvoRepository extends JpaRepository<RepOrderMvoEntity, Integer>, JpaSpecificationExecutor<RepOrderMvoEntity> {
+    <T> T findByRepIdAndVehicleOwner(Integer id, String vehicleOwner, Class<T> type);
```

Comment:
Can this not just be:
`List<RepOrderMvoEntity> findByRepIdAndVehicleOwner(Integer id, String vehicleOwner);`

Reply:
This will return the whole Entity but the implementation uses Projection and hence this is required